Dockerfile

```diff
@@ -23,6 +25,9 @@ RUN autoconf && ./configure && make && make install
 
 # Install Icarus verilog
 WORKDIR /home
+RUN apt-get update && apt-get install -y \
+  autoconf \
+  gperf
```

Comment:
Can you merge this with the `apt-get` command above? Each extra command in a dockerfile makes the final image bigger.